responses/cs/HassClimateGetTemperature.yaml

```diff
@@ -2,4 +2,4 @@ language: cs
 responses:
   intents:
     HassClimateGetTemperature:
-      default: '{{ state.state }} stupňů'
+      default: '{{ state.state }} {{ ''stupeň'' if state.state == 1 else ''stupně'' if state.state in [2,3,4] else ''stupňů'' }}'
```

Comment:
Official language would say also "21 stupeň", but for most of people "21 stupňů" sounds more natural.

For negative values we keep "stupňů" everywhere?